website/docs/installation/configuration.mdx

```diff
@@ -401,7 +400,7 @@ Requires authentik 2022.9
 Configure how many gunicorn worker processes should be started (see https://docs.gunicorn.org/en/stable/design.html).
 
 If running in Kubernetes, the default value is set to 2 and should in most cases not be changed, as scaling can be done with multiple pods running the web server.
-Otherwise, authentik will use 1 worker for each 4 CPU cores + 1 as a value below 2 workers is not recommended.
+Otherwise, authentik will use 1 worker for each 4 CPU cores + 1, because a value below 2 workers is not recommended.
```

Comment:
Suggestion: "Otherwise, authentik will use 1 worker for each 4 CPU cores + 1.  A value below 2 workers is not recommended."

Also suggestion: Say _why_.